Components/badges/badges.css

```diff
@@ -0,0 +1,78 @@
+.badges-container {
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+    padding: 1rem;
+    margin-bottom: 1.5rem;
+    flex-wrap: wrap;
+    box-shadow: 1px 1px 2px rgba(155, 146, 146, 0.993);
+    border-radius: 10px;
+    border: 2px solid var(--light-border-color);
+    background-color: rgba(224, 220, 220, 0.294);
+    min-height: 100%;
+    width: 100%;
+}
+
+.badge {
+    position: relative;
+    width: max-content;
+    margin-right: 1em;
+}
+
+.status-online {
+    background-color: var(--online-badge);
+    right: .9em;
+}
+
+.status-offline {
+    background-color: var(--offline-badge);
+    right: .1px;
+}
+
+.status-donotdisturb {
+    background-color: var(--dontdisturb-badge);
+    right: .2em;
+}
+
+.status-ideal {
+    background-color: var(--ideal-badge);
+    right: .3px;
```

Comment:
In the last 2 avatars (smaller ones) the badge seems to come almost inside the avatar. Try using the 'bottom' property.
It might also be a good practice in the long run to keep the value of 'right' property in the same unit for all badges, either rem or px.